src/components/Search/index.tsx

```diff
@@ -77,7 +78,7 @@ const Search = ({ opacity = false, content }: { opacity?: boolean; content?: str
     if (!query) {
       appContext.toastMessage('Please input valid content', 3000)
     } else {
-      fetchSearchResult(query)
+      fetchSearchResult(prefixHex(query))
```

Comment:
If the value is block number, decimal and hex data are both ok. 
So we can't add `0x` to every data, maybe we can distinguish according to the length of keyword.

Maybe we have a more better way to solve this question and you can think twice again.